src/eos-google-chrome-app

```diff
@@ -108,26 +110,7 @@ class GoogleChromeFlatpakLauncher:
             sandbox_process.communicate(input=os.linesep.encode())
             logging.info("Sandbox process stopped")
 
-    def _get_chrome_app_info(self):
-        app_lib_path = None
-        runtime_path = None
-
-        app = find_flatpak_ref(self._installation, self.FLATPAK_CHROME_APP_ID)
-        if not app:
-            logging.info("Chrome application is not installed")
-            return None
-
-        app_path = app.get_deploy_dir()
-        if not app_path or not os.path.exists(app_path):
-            exit_with_error("Could not find Chrome's application directory")
-
-        app_lib_path = os.path.join(app_path, 'files/lib/x86_64-linux-gnu')
-        if not os.path.exists(app_lib_path):
-            exit_with_error("Could not find bundled libraries for Chrome")
-
-        if app_path and not app_lib_path:
-            exit_with_error("Chrome's installation directory found, but not bundled libraries")
-
+    def _get_legacy_chrome_path(self, app_path, app_info):
```

Comment:
maybe a better name would be `_get_legacy_chrome_runtime_path()`, but no strong feelings